nautobot/dcim/filters/__init__.py

```diff
@@ -903,6 +904,11 @@ class DeviceFilterSet(
         to_field_name="version",
         label="Software version (version or ID)",
     )
+    access_point_group = NaturalKeyOrPKMultipleChoiceFilter(
+        queryset=AccessPointGroup.objects.all(),
+        to_field_name="name",
+        label="Access Point Group (name or ID)",
+    )
 
```

Comment:
Not a blocker here, maybe a follow-up FR, but a wireless networks filter would be nice to have on device and controller.